package.json

```diff
@@ -48,6 +48,8 @@
   "resolutions": {
     "@babel/runtime": "7.22.15",
     "@rollup/plugin-commonjs": "11.0.2",
+    "@tanstack/react-query": "5.29.2",
```

Comment:
forcing resolution at the root of the monorepo should be the last resort for fixing dependencies issues

doesn't updating tanstack query version to the same version in each packages solve the issue ?